app/src/androidTest/java/org/mozilla/reference/browser/ui/robots/BrowserRobot.kt

```diff
@@ -24,16 +26,18 @@ class BrowserRobot {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
         val testContent = mDevice.findObject((UiSelector().textContains(expectedText)))
 
-        mDevice.waitForIdle()
         testContent.waitForExists(waitingTime)
         assertTrue(testContent.exists())
     }
 
     fun verifyFXAUrl() {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-        val redirectUrl = "https://accounts.firefox.com/"
+        // if WIFI off
+        val redirectUrl = "https://github.com/login"
```

Comment:
This seems to fail intermittently, not sure why... 🤔

Reply:
Probably the website loading issue?

Reply:
@npark-mozilla on occasion it can't find this URL.  ultimately the redirect (if you're logged in to github), will redirect you to firefox accounts URL.  strangely, on one occasion when I wasn't logged into github, the redirect to the firefox accts URL took place anyway.  it 'may' be that the redirect has a bug?  not sure.  i'd have to spend some more time looking into this.

Reply:
perhaps in future we could use simpler website like example.com?